source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -297,7 +298,7 @@ def get_instances(self, name=None, lon=None, lat=None, object_id=None, api='ALL'
         else:
             # Requests without any coverage
             # fetch all the authorized instances (free + private) using cached function has_access()
-            authorized_instances = self._get_authorized_instances(user, api)
+            authorized_instances = get_all_available_instances(user)  # self._get_authorized_instances(user, api)
```

Comment:
I believe you can remove the comment. `git` history is enough to remember.

Reply:
:+1: